sdarot.py

```diff
@@ -17,12 +17,17 @@
 class SdarotPy:
     def __init__(self, sid, season_range=None, episode_range=None, output_path=Configuration.OUTPUT_PATH):
 
+        self.sid = sid
+
+        # get series name from the webpage
+        series_info = self.get_series_info()
+        self.series_name = series_info['name'];
         if season_range is None:
-            season_range = [1]
+            season_range = range(1, int(series_info['seasons']) + 1)
```

Comment:
seasons can start from 0, but its not that often so i don't really care

Reply:
i've fixed it to start from 0 and also fixed the crushing and added an exit to the selection